lib/std/time.zig

```diff
@@ -59,6 +59,398 @@ pub fn sleep(nanoseconds: u64) void {
     std.os.nanosleep(s, ns);
 }
 
+pub const TimeZone = @import("time/TimeZone.zig");
+
+pub const LocalTimeError = error{
+    /// The local time zone of the system could not be determined.
+    LocalTimeUnavailable,
+} || std.mem.Allocator.Error;
+
+pub fn localtime(allocator: std.mem.Allocator) LocalTimeError!TimeZone {
```

Comment:
Are you planning to support the Posix style footers and/or TZ environment variable? I have support for Posix style TimeZone specifications (https://codeberg.org/geemili/chrono-zig/src/branch/master/src/tz/Posix.zig) and support for loading from the TZ environment variable: https://codeberg.org/geemili/chrono-zig/src/branch/master/src/tz.zig#L140-L154